db/seeds.rb

```diff
@@ -5,3 +5,24 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+Image.create!(imageurl: 'https://ichef.bbci.co.uk/news/660/cpsprodpb/37B5/production/_89716241_thinkstockphotos-523060154.jpg')
```

Comment:
Great use of `create!`. That said there's a cool trick you can use so that you only have to call `create!` once rather than 20 times. Take another pass at it and let me know what you discover!

Reply:
I found that you can initialize the Image using an array, like:
```
Image.create!([
  { imageurl: 'https://ichef.bbci.co.uk/news/660/cpsprodpb/37B5/production/_89716241_thinkstockphotos-523060154.jpg' },
  { imageurl: 'https://www.newton.ac.uk/files/covers/968361.jpg' },
  ...
])
```
Is this syntax preferred?

Reply:
Yep!